### ISSUE TITLE
Metrics instrumentation sqlite3

### ISSUE THREAD

**Issue by srikanthccv** [open-telemetry/opentelemetry-python-contrib#1147]
DB semconv : https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md

**Comment by artemrys** [comment]
I would like to help with this one, I am a newbie in the open-telemetry world, but I see you marked this one as "good first issue", so can you please assign this one to me?

**Comment by srikanthccv** [comment]
Thanks. Feel free to ask any questions here or on our slack channel.

**Comment by artemrys** [comment]
Moving here a conversation from Slack:

It seems that there is no work that needs to be done in `sqlite3` package specifically, as there is [no signs of useful attributes](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/a3f60a65926fce2d2a1531aae7dfeb1a5961248c/instrumentation/opentelemetry-instrumentation-sqlite3/src/opentelemetry/instrumentation/sqlite3/__init__.py) for `sqlite3` connection object that could be extracted according to [DB semconv](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md).

@srikanthccv said

> If there are no attributes that's fine but instrumentation should still be done just like how tracing instrumentation done. Looks like all the work is done in dbapi instrumentation package. You can work on that package.

@srikanthccv do you know is there any example that already implements something similar?

**Comment by lzchen** [comment]
@artemrys 
> do you know is there any example that already implements something similar?

I do not believe metrics have been implemented for any db instrumentations in Python.

**Comment by lzchen** [comment]
@Annosha 

Yup, can reassign to you.
